Remove the tests for the Dashing branch.

clalancette opened the pull request:
In dashing launch_testing doesn't have hooks for pytest, and
it is too much work to fix.  So just remove the tests in Dashing;
we'll still have them for Eloquent and forwards on the other
branches.

Signed-off-by: Chris Lalancette <clalancette@openrobotics.org>

sloretz commented on joint_state_publisher/package.xml:
```diff
@@ -18,12 +18,6 @@
   <exec_depend>std_msgs</exec_depend>
 
-  <test_depend>ament_copyright</test_depend>
-  <test_depend>ament_xmllint</test_depend>
```
I think `ament_copyright` and `ament_xmllint` tests still work. I would recommend keeping them. It's just the launch tests that are broken.

clalancette replied:
Done in b882325

sloretz replied:
Almost done :wink: The `test_copyright.py` and `test_xmllint.py` files need to be un-deleted. Maybe the `tests_require=['pytest'],` line in `setup.py` is needed too, but I'm not 100% sure.

clalancette replied:
Bah, of course.  Sorry, went too fast, and we obviously need that.  Fixed and force-pushed now, thanks.

clalancette committed:
Remove the tests for the Dashing branch.

In dashing launch_testing doesn't have hooks for pytest, and
it is too much work to fix. So just remove the tests in Dashing;
we'll still have them for Eloquent and forwards on the other
branches.

Signed-off-by: Chris Lalancette <clalancette@openrobotics.org>
